manifests/init.pp

```diff
@@ -140,6 +140,18 @@
                 warning('Packages for Ruby 2.1 are not available from default repositories.')
               }
             }
+            /^2\.2.*$/:{
```

Comment:
That means there should also be checks to ensure that 2.2.1 does a thing and 2.20.1 does not do that thing.